lib/grpc/secureconn/factory_test.go

```diff
@@ -80,7 +80,7 @@ func TestSecureConnFactory(t *testing.T) {
 
 		s, ok := status.FromError(err)
 		require.True(t, ok)
-		assert.Regexp(t, "authentication handshake failed: x509.*valid for root-a-service, not root-foo-service", s.Message())
+		assert.Regexp(t, "authentication handshake failed: x509.*valid for root-a-service, root-a-service, not root-foo-service", s.Message())
```

Comment:
why does this need to change ?

Reply:
Golang 1.19 onwards, the certificate won't be valid if they do not have `Subject Alternative Name`. We have generated the new cert (lib/grpc/secureconn/testdata/root-a-service.crt) with SAN having more than one DNS. That's why we need to modify the assertion stmt.